contents/handbook/company/security.md

```diff
@@ -64,15 +64,15 @@ We have reviewed our architecture, data flows and agreements to ensure that our
 
 PostHog does not require personally identifiable information or personal data to perform product analytics, and we provide extensive controls for customers wishing to minimize personal data collection from their end users. We provide separate guidance for our customers on how to use PostHog in a GDPR-compliant way in our [Docs](/docs/integrate/gdpr). 
 
-Additional measures:
+Technical and Organizational Measures ('TOMs')
 
-- We enter into Data Process Agreements (DPAs) with PostHog Cloud customers when requested - [our standard agreement is here](https://docs.google.com/document/d/1xfpP1SCFoI1qSKM6rEt9VqRLRUEXiKj9_0Tvv2mP928/edit?usp=sharing). We maintain a register of all DPAs we have entered into. 
```

Comment:
@charlescook-ph could we change the word customers to subscribers?  Is it feasible to only do DPAs for paying customers?

Reply:
'Customers' still covers people paying $0 to be fair, I think it's just semantics here but feel free to change if you think it matters!